src/components/StyledTabs.js

```diff
@@ -0,0 +1,14 @@
+// import * as React from "react";
+// import Tabs from "@material-ui/core/Tabs";
+// import { styled } from '@material-ui/core/styles';
+
+// // custom style for tab bar appearance
+// const StyledTabs = styled(Tabs)
+//     ({
+//         // change appearance of tab bar- current tab indicator
+//         "& .MuiTabs-indicator": {
+//             backgroundColor: `${pokemonTypeColor}`
+//         }
+//     });
+
+// export default StyledTabs;
```

Comment:
Do you guys still need this commented code?

Reply:
Had no idea this was in, garrett just mentioned it! Deleting it and pushing up again as we speak.